components/ingress-setup-image/Dockerfile.base.arm64

```diff
@@ -0,0 +1,39 @@
+FROM docker:17.12.0-ce as static-docker-source
```

Comment:
I believe this docker image is specific to setting up IAP on GCP.
So I'm not really sure it makes sense to provide an arm64 variant.

Reply:
Hi @jlewi , thanks for reviewing! The reason why I added this base image is that `google/cloud-sdk:alpine` is an x86_64 only image. In order to build this `ingress-setup-image` on arm64, I have to add this google cloud base image and use `docker --buildarg` to avoid breaking existing x86_64 building strategy.

My previous idea before our discussion in [kubeflow#2337](https://github.com/kubeflow/kubeflow/issues/2337) was to support as much component as I could. So if you think it is unnecessary to add arm64 support for this `ingress-setup-image`, I would also happy to remove this.

Reply:
Yes. Please remove. This image is only used for GCP. If you are on GCP you aren't running on Arm64 so there's no reason to maintain an Arm64 variant of this image.

Reply:
Thanks for the explanation @jlewi. The `ingress-setup-image` code has been reverted.